engine/flink/generic/src/test/scala/pl/touk/nussknacker/genericmodel/GenericItSpec.scala

```diff
@@ -321,14 +322,14 @@ class GenericItSpec extends FunSuite with FlinkSpec with Matchers with KafkaSpec
       ConfluentSchemaRegistryProvider(factory, processObjectDependencies)
   }
 
-  private var registrar: FlinkStreamingProcessRegistrar = _
+  private var registrar: FlinkProcessRegistrar = _
   private lazy val valueSerializer = new KafkaAvroSerializer(schemaRegistryMockClient)
   private lazy val valueDeserializer = new KafkaAvroDeserializer(schemaRegistryMockClient)
 
   override protected def beforeAll(): Unit = {
     super.beforeAll()
     val modelData = LocalModelData(config, creator)
-    registrar = FlinkStreamingProcessRegistrar(new FlinkProcessCompiler(modelData), config, ExecutionConfigPreparer.unOptimizedChain(modelData, None))
+    registrar = process.registrar.FlinkProcessRegistrar(new FlinkProcessCompiler(modelData), config, ExecutionConfigPreparer.unOptimizedChain(modelData, None))
```

Comment:
Why `process.registrar.` instead simply `FlinkProcessRegistrar`? :)

Reply:
it was some stupid auto refactor I think